src/hooks/useProfiles.ts

```diff
@@ -24,27 +17,14 @@ export const useSelectProfile = () => {
     onMutate: ({ avatarUrl }) => {
       useProfileStore.setState({ selectingProfileAvatar: avatarUrl });
     },
-    onSuccess: async (response) => {
-      const profile = response?.responseData;
-      if (profile?.credentials?.access_token) {
-        persist.setItem(PERSIST_PROFILE, profile);
-        persist.setItemStorage('inplayer_token', {
-          expires: profile.credentials.expires,
-          token: profile.credentials.access_token,
-          refreshToken: '',
-        });
-        useFavoritesStore.setState({ favorites: [] });
-        useWatchHistoryStore.setState({ watchHistory: [] });
-        useProfileStore.setState({ profile });
-        await initializeAccount().finally(() => {
-          useProfileStore.setState({ selectingProfileAvatar: null });
-          navigate('/');
-        });
-      }
+    onSuccess: async () => {
+      useProfileStore.setState({ selectingProfileAvatar: null });
+      navigate('/');
     },
     onError: () => {
       useProfileStore.setState({ selectingProfileAvatar: null });
-      throw new Error('Unable to enter profile.');
+      navigate('/u/profiles');
+      console.error('Unable to enter profile.');
```

Comment:
This is only useful for dev?

```suggestion
      logDev('Unable to enter profile.');
```